### PR TITLE
Add known issue for radio second port

### LINKED ISSUE #2207: List of known issues (for 2023) could be expanded
This comment is about the "known issues" page https://docs.wpilib.org/en/stable/docs/yearly-overview/known-issues.html

1. The 2023 known issues page does explain that the Driver Station may not detect joysticks at startup, and suggests that a workaround is to rescan for joysticks once. This works for joysticks, but not always for controllers, which may require to rescan twice as explained at https://www.chiefdelphi.com/t/commandxboxcontroller-lefttrigger-and-righttrigger-not-triggering/427504

Please add a paragraph explaining that controllers may be detected as DirectInput devices rather than XInput devices after a single refresh, and detail how one can check if a controller was detected as a DirectInput rather than an XInput device in the Driver Station or programmatically.

2. Also, while totally unrelated to the problem above, it should be noted that this page makes no mention of the problem with the second port of the OpenMesh radio described at https://www.chiefdelphi.com/t/why-you-probably-shouldnt-use-the-second-port-on-your-openmesh-om5p-radio-and-embrace-using-an-ethernet-switch-instead/406374

If the problem with the second port of the OpenMesh radio is acknowledged, then it would also be good to mention it on this page.

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #2207

### CODE REVIEW

**Comment by Daltz333** [comment]
Since this is a permanent issue, can we lower it down on the page hierarchy?

File: source/docs/yearly-overview/known-issues.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ This article details known issues (and workarounds) for FRC\ |reg| Control Syste
 Open Issues
 -----------
 
+
```
Comment (sciencewhiz) [inline source/docs/yearly-overview/known-issues.rst:11]:
```suggestion
```

**Comment by sciencewhiz** [comment]
I moved it down below the other issues recently added, but consequently it should be merged quickly or else it will be wierd to merge something in the middle. It would naturally move down next year as issues are fixed and new issues are added

**Review by Daltz333 — APPROVED** [review]

File: source/docs/yearly-overview/known-issues.rst [hunk 2]
```diff
@@ -24,6 +24,13 @@ Driver Station Reporting No Code
 
  .. note:: Pressing the physical :guilabel:`User` button on the roboRIO for 5 seconds can also cause the robot code to not start, but a reboot will not start the robot code. If the robot code does not start after rebooting, press the :guilabel:`User` button. Ensure that nothing on the robot is in contact with the :guilabel:`User` button.
 
+Radio Second Port Sometimes Fails to Communicate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Issue:** There is a rare occurrence in the OM5P Radios that causes the second Ethernet port (the one farthest from the power plug) to not communicate.
+
+**Workaround:** Generally, power cycling the radio will restablish communication with the second port. Alternately, utilize a network switch such as the tp-link switch available from `FIRST Choice <https://firstchoicebyandymark.com/fc-cn-9024>`__ or the `brainboxes SW-005 <https://www.brainboxes.com/product/industrial-ethernet-switches/fast-ethernet/sw-005>`__ and plug all ethernet devices into the network switch and then plug the switch into the radio's first Ethernet port. This also allows easier tethering while at competition.
```
Comment (Starlight220) [inline source/docs/yearly-overview/known-issues.rst:32]: Is this the correct spelling for "brainboxes"

Comment (sciencewhiz) [reply source/docs/yearly-overview/known-issues.rst:32]: yes, see the link

Comment (Starlight220) [reply source/docs/yearly-overview/known-issues.rst:32]: Sorry, I meant capitalization, not spelling. But that seems to be correct as well.

**Review by Starlight220 — APPROVED** [review]

**Pull request merged**